editor/editor_inspector.cpp

```diff
@@ -489,6 +489,45 @@ StringName EditorProperty::_get_revert_property() const {
 	return property;
 }
 
+void EditorProperty::_update_property_bg() {
+	// This function is to be called on EditorPropertyResource, EditorPropertyArray and EditorPropertyDictionary.
+	// Behaviour is undetermined on any other EditorProperty.
+	if (!is_inside_tree()) {
+		return;
+	}
+
+	updating_theme = true;
+
+	if (bottom_editor) {
+		ColorationMode coloration_mode = (ColorationMode)(int)EDITOR_GET("interface/inspector/coloration_mode");
+		bool delimitate_all_container_and_resources = (bool)EDITOR_GET("interface/inspector/delimitate_all_container_and_resources");
```

Comment:
```suggestion
		bool delimitate_all_container_and_resources = EDITOR_GET("interface/inspector/delimitate_all_container_and_resources");
```
No need to cast when assigning.